documentation/docs/03-template-syntax/14-in-and-out.md

```diff
@@ -5,6 +5,10 @@ title: in: and out:
 The `in:` and `out:` directives are identical to [`transition:`](transition), except that the resulting transitions are not bidirectional — an `in` transition will continue to 'play' alongside the `out` transition, rather than reversing, if the block is outroed while the transition is in progress. If an out transition is aborted, transitions will restart from scratch.
 
 ```svelte
+<script>
+  import { fade, fly } from 'svelte/transition';
```

Comment:
if we have a `<script>` for the imports, we also need to declare `visible` since it's no longer implicit

```suggestion
  import { fade, fly } from 'svelte/transition';
  
  let visible = $state(false);
```